en/uavcan/ark_flow.md

```diff
@@ -94,4 +94,4 @@ Then and clone the sources from https://github.com/PX4/PX4-Autopilot using *git*
 cd PX4-Autopilot
 make ark_can-flow
 ```
-Then follow instructions for [UAVCAN firmware updating](../uavcan/node_firmware.md) instructions using the binary located in build/ark_can-flow_default named XX-X.X.XXXXXXXX.uavcan.bin
\ No newline at end of file
+Then follow instructions for [UAVCAN firmware updating](node_firmware.md) instructions using the binary located in build/ark_can-flow_default named XX-X.X.XXXXXXXX.uavcan.bin
```

Comment:
Nearly there. So I think perhaps if you change this section heading from ##Build to ##Firmware update or ##Building Firmware. Add a note explaining when this needs to be done. My understanding is that this would be optional/rare. So something like "Occasionally we update the Ark Flow firmware. You can build and update to the latest at your convenience ..."

The point is that you don't want most users to have to think about firmware updates for the Ark. But those who do need to can.

Reply:
I changed it to Building Firmware for Development